tests/e2e/utils/auto-fsx-setup.py

```diff
@@ -174,7 +174,7 @@ def create_fsx_iam_service_account():
             "--cluster",
             CLUSTER_NAME,
             "--attach-policy-arn",
-            FSX_IAM_POLICY_NAME,
+            FSX_IAM_POLICY_ARN,
```

Comment:
how did it work before ?

Reply:
This worked because it was running on an existing cluster and since the FSX_IAM_POLICY_NAME was fixed, it already existed. I had an action item to run FSx on a new cluster and thus caught this issue. 
Thus I made `FSX_IAM_POLICY_NAME ` it variable now such that these issues don't slip through.